ldaptor/protocols/ldap/ldapsyntax.py

```diff
@@ -247,7 +247,7 @@ def toWire(self):
         lst = list(self.items())
         lst.sort()
         for key, values in lst:
-            if key != "objectClass":
+            if key not in self._object_class_keys:
```

Comment:
just above "objectClass" is handled explicitly too, does that need to use _object_class_keys ?

```
        objectClasses = list(self.get("objectClass", []))
        objectClasses.sort()
        a.append(("objectClass", objectClasses))
```